src/simplejavacalculator/Calculator.java

```diff
@@ -78,6 +85,14 @@ public Double calculateMono(MonoOperatorModes newMode, Double num) {
         if (newMode == MonoOperatorModes.tan) {
             return Math.tan(num);
         }
+        if (newMode == MonoOperatorModes.log) {
+            return log10(num);
+        }
+        if (newMode == MonoOperatorModes.rate) {
+           return num/100;
```

Comment:
I know, its a small detail, but for consistency and standard, can you add space between the `/`
```java
return num / 100;
```

Thanks!